modules/op-release-notes-1-12.adoc

```diff
@@ -0,0 +1,192 @@
+== Fixed issues
+
+* Before this update, {pac} ran pipeline runs based only on branch base names, and could  incorrectly trigger pipeline runs with the same base name but different branch name. With this update, {pac} does checks both the base name and the exact branch name of a pipeline run.
+
+* Before this update, an incoming webhook event could trigger multiple pipeline runs configured for other events. With this update, an incoming webhook event triggers only a pipeline run configured for the webhook event.
+
+* With this update, the `pac-gitauth` secrets are now explicitly deleted when cleaning up a pipeline run, in case the `ownerRef` on the pipeline run gets removed.
+
+* Before this update, when a task in a pipeline run failed with a reason message, the entire pipeline run failed with a `PipelineValidationFailed` reason. With this update, the pipeline run fails with the same reason message as the task that failed.
+
+*  Before this update, the `disable-ha` flag was not correctly passed to the Pipelines controller, and the high availability (HA) functionality was never enabled. With this update, the HA functionality is enabled if you set the `disable-ha` flag to `false`.
+
+* Before this update, the `skopeo-copy` cluster task would fail when attempting to copy images mentioned in  ConfigMap data. With this update, the `skopeo-copy` cluster task completes properly.
+
+* With this update, a pipeline run automatically generated by the `tkn pac generate –language=java` command has correct annotations and parameter names.
+
+* Before this update, only a user with the administrative permissions could successfully run the `tkn pac create repository` command. With this update, any authorized user can run the `tkn pac create repository` command.
+
+* Before this update, the `/test <run-name>` and `/retest <run-name>` user comments, which specified a particular pipeline, did not trigger pipeline runs as expected. With this update, these comments trigger pipeline runs successfully.
+
+* Before this update, if there were multiple pipeline runs in the `.tekton` folder with the `generateName` field and not the `Name` field, the pipeline runs failed. This update fixes the issue.
+
+* Before this update, in {pac} when using GitLab, a pipeline run was triggered by any event in a merge request, including adding labels and setting status. With this update, the pipeline run is triggered only when there is an open, reopen, or push event. A comment containing the status of the checks is now posted on the merge request.
+
+* Before this update, while a pipeline run was waiting for approval, the status of the check could be displayed as `skipped`. With this update, the correct `pending approval` status is displayed.
+
+* Before this update, the bundles resolver sometimes set the type to `Task` when attempting to retrieve a pipeline, leading to errors in retrieval. With this update, the resolver uses the correct type to retrieve a pipeline.
```

Comment:
```suggestion
* Before this update, the bundle resolver sometimes set the type to `Task` when attempting to retrieve a pipeline, leading to errors in retrieval. With this update, the resolver uses the correct type to retrieve a pipeline.
```

Reply:
It is "bundles resolver" https://docs.openshift.com/pipelines/1.11/create/remote-pipelines-tasks-resolvers.html#resolver-bundles-config_remote-pipelines-tasks-resolvers and I can not rename it in docs because one configuers it as bundles in the YAML